examples/operator-rbac/1-setup-operator-rbac.ipynb

```diff
@@ -0,0 +1,753 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Feast Operator with RBAC Configuration\n",
+    "## Objective\n",
+    "\n",
+    "This demo provides a reference implementation of a runbook on how to enable Role-Based Access Control (RBAC) for Feast using the Feast Operator with the Kubernetes authentication type.\n",
+    "\n",
+    "The demo steps include deploying the Feast Operator, creating Feast instances with server components (registry, offline store, online store), and Feast client testing locally. The goal is to ensure secure access control for Feast instances deployed by the Feast Operator.\n",
+    " \n",
+    "Please read these reference documents for understanding the Feast RBAC framework.\n",
+    "- [RBAC Architecture](https://docs.feast.dev/v/master/getting-started/architecture/rbac) \n",
+    "- [RBAC Permission](https://docs.feast.dev/v/master/getting-started/concepts/permission).\n",
+    "- [RBAC Authorization Manager](https://docs.feast.dev/v/master/getting-started/components/authz_manager)\n"
+   ]
+  },
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": [
+    "## Deployment Architecture\n",
+    "In this notebook, we will deploy a distributed topology of Feast services, which includes:\n",
+    "\n",
+    "* `Registry Server`: Handles metadata storage for feature definitions.\n",
+    "* `Online Store Server`: Uses the `Registry Server` to query metadata and is responsible for low-latency serving of features.\n",
+    "* `Offline Store Server`: Uses the `Registry Server` to query metadata and provides access to batch data for historical feature retrieval.\n",
+    "* `Kubernetes` Authentication types for RBAC Configuration for Feast resources.\n",
```

Comment:
The last 2 bullets doesn't seem to be related to "distributed topology of Feast services".
The last one in particular, it looks like it's an objective of the notebook rather any architectural component.

Reply:
make sense, updated